8-7 Release Notes (#4007)
astronomer/docs: astro/release-notes.md

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

> ## August 6, 2024

Author: ```suggestion
## August 7, 2024
```

### Customize labels on your metrics export

You can now add Key:Value pair labels to metrics that you export using the Universal Metrics Exporter. This allows you to tag and record metrics coming from specific Deployments or Workspaces. See [Export metrics from Astro](https://www.astronomer.io/docs/astro/export-metrics) for more information.

### Additional improvements

> - Added a new template demonstrating how to run dbt projects on Astro using [Cosmos](https://www.astronomer.io/integrations/dbt/) when you're onboarding. See [Get started on Astro](https://cloud.astronomer.io/start-astro) to set up a trial account and try a template.

Reviewer: ```suggestion
- Added a new template demonstrating how to run dbt projects on Astro using [Cosmos](https://www.astronomer.io/integrations/dbt/) when you're onboarding. See [Get started on Astro](https://cloud.astronomer.io/start-astro) to set up a trial account and try out a template.
```

Reviewer: thought it sounded like a word was missing here?

> - Added a new template allowing you to deploy dbt on Astro when you're onboarding. See [Deploy dbt projects on Astro](https://www.astronomer.io/docs/astro/deploy-dbt-project) for information about dbt deploys, or go to [Get started on Astro](https://cloud.astronomer.io/start-astro) to set up a trial account and try a template.

Reviewer 2: This new template appears to be more about demonstrating how to run dbt projects using Cosmos rather than deploying with dbt deploys. We only support dbt deploys for dbt projects outside of the Astro project, but in this template it is within it, so we might not want to refer to dbt deploys here.

> - Added a new template allowing you to deploy dbt on Astro when you're onboarding. See [Deploy dbt projects on Astro](https://www.astronomer.io/docs/astro/deploy-dbt-project) for information about dbt deploys, or go to [Get started on Astro](https://cloud.astronomer.io/start-astro) to set up a trial account and try a template.

Author: ```suggestion
- Added a new template demonstrating how to run dbt projects on Astro using [Cosmos](https://www.astronomer.io/integrations/dbt/) when you're onboarding. See [Get started on Astro](https://cloud.astronomer.io/start-astro) to set up a trial account and try a template.
```
